examples/custom_encoder/qwen3_5_vision_encoder.py

```
@@ -0,0 +1,139 @@
+        )
+        self._visual = model.model.visual.eval().to(self._device)
+        model.model.visual = None
+        del model
+        gc.collect()
+
+    def preprocess(self, raw: str) -> Preprocessed[Qwen35ImageInputs]:
+        """Fetch, decode, and patchify one image on a preprocessing thread."""
+        if self._processor is None:
+            raise RuntimeError("Qwen35VisionEncoder is not loaded")
+
+        with urllib.request.urlopen(raw, timeout=30) as response:
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)